cloudbuild.yaml

```diff
@@ -155,30 +155,30 @@ steps:
     #!/bin/sh
     set -o errexit
     set -o xtrace
-    docker tag bazel/experimental/nodejs:nodejs14_debian9        gcr.io/$PROJECT_ID/nodejs:latest
-    docker tag bazel/experimental/nodejs:nodejs10_debian9        gcr.io/$PROJECT_ID/nodejs:10
-    docker tag bazel/experimental/nodejs:nodejs12_debian9        gcr.io/$PROJECT_ID/nodejs:12
-    docker tag bazel/experimental/nodejs:nodejs14_debian9        gcr.io/$PROJECT_ID/nodejs:14
-    docker tag bazel/experimental/nodejs:nodejs14_debug_debian9  gcr.io/$PROJECT_ID/nodejs:debug
-    docker tag bazel/experimental/nodejs:nodejs10_debug_debian9  gcr.io/$PROJECT_ID/nodejs:10-debug
-    docker tag bazel/experimental/nodejs:nodejs12_debug_debian9  gcr.io/$PROJECT_ID/nodejs:12-debug
-    docker tag bazel/experimental/nodejs:nodejs14_debug_debian9  gcr.io/$PROJECT_ID/nodejs:14-debug
-    docker tag bazel/experimental/nodejs:nodejs14_debian9        gcr.io/$PROJECT_ID/nodejs-debian9:latest
-    docker tag bazel/experimental/nodejs:nodejs10_debian9        gcr.io/$PROJECT_ID/nodejs-debian9:10
-    docker tag bazel/experimental/nodejs:nodejs12_debian9        gcr.io/$PROJECT_ID/nodejs-debian9:12
-    docker tag bazel/experimental/nodejs:nodejs14_debian9        gcr.io/$PROJECT_ID/nodejs-debian9:14
-    docker tag bazel/experimental/nodejs:nodejs14_debug_debian9  gcr.io/$PROJECT_ID/nodejs-debian9:debug
-    docker tag bazel/experimental/nodejs:nodejs10_debug_debian9  gcr.io/$PROJECT_ID/nodejs-debian9:10-debug
-    docker tag bazel/experimental/nodejs:nodejs12_debug_debian9  gcr.io/$PROJECT_ID/nodejs-debian9:12-debug
-    docker tag bazel/experimental/nodejs:nodejs14_debug_debian9  gcr.io/$PROJECT_ID/nodejs-debian9:14-debug
-    docker tag bazel/experimental/nodejs:nodejs14_debian10       gcr.io/$PROJECT_ID/nodejs-debian10:latest
-    docker tag bazel/experimental/nodejs:nodejs10_debian10       gcr.io/$PROJECT_ID/nodejs-debian10:10
-    docker tag bazel/experimental/nodejs:nodejs12_debian10       gcr.io/$PROJECT_ID/nodejs-debian10:12
-    docker tag bazel/experimental/nodejs:nodejs14_debian10       gcr.io/$PROJECT_ID/nodejs-debian10:14
-    docker tag bazel/experimental/nodejs:nodejs14_debug_debian10 gcr.io/$PROJECT_ID/nodejs-debian10:debug
-    docker tag bazel/experimental/nodejs:nodejs10_debug_debian10 gcr.io/$PROJECT_ID/nodejs-debian10:10-debug
-    docker tag bazel/experimental/nodejs:nodejs12_debug_debian10 gcr.io/$PROJECT_ID/nodejs-debian10:12-debug
-    docker tag bazel/experimental/nodejs:nodejs14_debug_debian10 gcr.io/$PROJECT_ID/nodejs-debian10:14-debug
+    docker tag bazel/nodejs:nodejs14_debian9        gcr.io/$PROJECT_ID/nodejs:latest
+    docker tag bazel/nodejs:nodejs10_debian9        gcr.io/$PROJECT_ID/nodejs:10
+    docker tag bazel/nodejs:nodejs12_debian9        gcr.io/$PROJECT_ID/nodejs:12
+    docker tag bazel/nodejs:nodejs14_debian9        gcr.io/$PROJECT_ID/nodejs:14
```

Comment:
Yea, I was thinking the same thing, but I kept it at debian9 bc of backwards compatibility at the time, but the more I think about it, the more I tend to moving to debian10 as default. @chanseokoh thoughts?

Reply:
I wish we could advance the Distroless base from Debian 9 to 10. There are some blocking issues because of the Java images where Debian 10 doesn't provide Java 8. We do need to switch to 10 sooner than later, but for now, I think we will stay with Debian 9. At least we won't switch the base in this PR.